Sources/Helpers/OAAppSettings.m

```diff
@@ -958,7 +958,7 @@ - (void)setValueFromString:(NSString *)strValue appMode:(OAApplicationMode *)mod
 
 - (NSString *)toStringValue:(OAApplicationMode *)mode
 {
-    return [NSString stringWithFormat:@"%.1f", [self get:mode]];
+    return [NSString stringWithFormat:@"%.2f", [self get:mode]];
```

Comment:
Why did you decide it is ok to change this? This method should only be used for import/export the osf file. It sets 2 floating points to make osf identical with Android.